file_formats/ie_formats/pro_v1.htm

```diff
@@ -130,19 +131,204 @@
               <li>Bit 0: Bounce from walls</li>
               <li>Bit 1: Pass target</li>
               <li>Bit 2: Draw centre VVC once</li>
-              <li>Bit 3: Hit immediately</li>
-              <li>Bit 4: Face target</li>
-              <li>Bit 5: Curved path</li>
+
+              <li>
+                Bit 3: Hit immediately
+                <ul>
+                  <li>Projectile Type (@ 0x8) must be either (2|Single Target) or (3|Area Effect).</li>
+                  <li>Travelling Projectile hits instantly, but is still displayed momentarily.</li>
+                  <li>Area Effect Explosion will be slightly off center of the target('s/ed) location.</li>
+                  <li>Compatibility: Suppressed by Extended Flag(BIT14|Lined Up Aoe)</li>
+                  <li>Suppresses: Extended Flags(BIT14|Lined up AoE)</li>
+                </ul>
+              </li>
+
+              <li>
+                Bit 4: Face target
+                <ul>
+                  <li>Dependency: Activated by Area Flag(BIT10|Draw Animation)</li>
+                  <li>
+                    Allows the (0x21C|Explosion Animation) to be oriented in the direction of casting (the VVC and its BAM must be setup to do so as well).
+                    <ul>
+                      <li>This requires the (0x21C|Explosion Animation) to include a VVC (VEF optional), not just a BAM.</li>
+                      <li>Any mirroring must be specified in the VVC.</li>
+                      <li>The <a href="../ie_formats/vvc_v1.htm">VVC</a> file's (0x40|Travel Orientation) must use (BIT0|Face Target).</li>
+                    </ul>
+                  </li>
+                </ul>
+              </li>
+
+              <li>
+                Bit 5: Curved path (Unknown – still "not implemented" on EE games)
```

Comment:
FYI, I suspect this was meant for magic missile, but then got obviated by the the other field.

Reply:
OK, will add.

Reply:
No need, it's just speculation.